CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [11.0.0] IN PROGRESS
 
+* Don't display affiliations of users with types `patron` or `dbc`. Refs UIU-2967.
+
```

Comment:
This one seems to be a target for Poppy bugfix. Please update the version in progress accordingly

Reply:
Generally, our practice is to target all changes to the master branch and then to cherry-pick fixed onto patch branches. If somebody else's PR hits master before yours lands, you'll have to do this anyway, so I just plan for things to work out this way. I would do it like this:
1. Bump the version on master in `package.json` to 10.1.0. You'll save a PR later if you do it in a separate PR now, but it's up to you.
2. Merge this PR.
3. Split a release branch named `b10.0` from the commit where master was tagged and push it up so others can also pick bug fixes onto : (`get checkout master; git checkout -b b10.0 v10.0.0; git push -u origin head`). 
4. If you bumped the version in step 1 in a separate PR, you can just pick the commit from this PR onto your release branch: `git cherry-pick -x SOME_HASH; git push`, you're done! If you bumped the version in step 1 in this PR, you'll need to split a branch from `b10.0`, pick this fix onto it, fix the version in `package.json`, push the branch and open a PR with a target of `b10.0`.